pandas/core/frame.py

```diff
@@ -9347,8 +9347,10 @@ def pivot(
             on the rows and columns.
         dropna : bool, default True
             Do not include columns whose entries are all NaN. If True,
-            rows with a NaN value in any column will be omitted before
-            computing margins.
+            - rows with a NA value in any column will be omitted before computing
+            margins,
+            - index/column keys containing NA values will be dropped (see ``dropna``
+            parameter in :method:``DataFrame.groupby``).
```

Comment:
I don't believe this renders as intended.

![image](https://github.com/user-attachments/assets/15146979-e8f7-4fe1-94b2-9702e3211e36)